packages/lit-html/src/lit-html.ts

```diff
@@ -124,7 +124,7 @@ const isArray = Array.isArray;
 const isIterable = (value: unknown): value is Iterable<unknown> =>
   isArray(value) ||
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  (value && typeof (value as any)[Symbol.iterator] === 'function');
+  ((value && typeof (value as any)[Symbol.iterator] === 'function') as boolean);
```

Comment:
I wonder if we should write this as `typeof (value as any)?.[Symbol.iterator] === 'function'`

Reply:
Done and this removed the need for the cast.